src/routes/dish.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+const dishController = require('../controllers/dishController');
+
+router.use(express.json());
+
+router.post('/', dishController.createDish); // Create a route for creating a new dish
+router.get('/', dishController.getAllDishes); // Create a route for getting all dishes
```

Comment:
We don't need those comments

Reply:
ok, I can delete it.

Reply:
These comments are for illustration only

Reply:
The code can explain itself.